tools/mocha/src/index.js

```diff
@@ -12,130 +12,22 @@ if(process.env.LONGJOHN)
 var _ = require('underscore');
 var path = require('path');
 var util = require('util');
-var smap = require('source-map');
-var csmap = require('convert-source-map');
 var textcov = require('./textcov.js');
 var istanbul = require('istanbul');
-var Mocha = require('mocha');
 var fs = require('fs');
 var tty = require('tty');
 var commander = require('commander');
+var childProcess = require('child_process');
+var async = require('async');
 
-var map = _.map;
-var values = _.values;
 var contains = _.contains;
 var memoize = _.memoize;
+var extend = _.extend;
 
-/* eslint no-process-exit: 1 */
+/* eslint no-process-exit: 0 */
```

Comment:
No, this actually disables the check, which is by default on. If we remove this line, eslint will start complaining.